pkg/api/validation/validation_test.go

```diff
@@ -39,6 +40,8 @@ func TestNilPath(t *testing.T) {
 }
 
 func TestNameFunc(t *testing.T) {
+	const nameRulesMessage = `must match the regex [a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)* (e.g. 'example.com')`
```

Comment:
Yeah

On Wed, Jul 6, 2016 at 3:28 PM, David Eads <notifications@github.com> wrote:

> In pkg/api/validation/validation_test.go
> <https://github.com/openshift/origin/pull/9578#discussion_r69796041>:
>
> > @@ -39,6 +40,8 @@ func TestNilPath(t *testing.T) {
> >  }
> >
> >  func TestNameFunc(t *testing.T) {
> > +	const nameRulesMessage = `must match the regex [a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)* (e.g. 'example.com')`
>
> Wow what a useful message. a-9,0-9,'.',- must start with a letter was too
> readable?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9578/files/dbd276c7f24e8896648fc10e38dc156293e69841..8a558b905f5d9772d4881afbaf84cd7339c7c90c#r69796041>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_pxOpaPYpqxJwFh7Kpl4dgkwP0BMfks5qTAHrgaJpZM4I-xTE>
> .
>